packages/elements/src/react/common/form/index.tsx

```diff
@@ -554,8 +565,9 @@ function FieldState({ children }: FieldStateRenderFn) {
 
   const message = feedback?.message instanceof ClerkElementsFieldError ? feedback.message.message : feedback?.message;
   const codes = feedback?.codes;
+  const config = feedback && 'config' in feedback ? feedback?.config : undefined;
```

Comment:
Maybe I'm missing something, but what does this config achieve ? Should this be fetched from the environment

Reply:
Alex and I thought the same thing, but the config is currently at `Clerk.__unstable__environment.userSettings.passwordSettings`, so we felt it was better to provide it as a prop rather than have users explicitly access an unstable prop in their code.

Reply:
The config contains important values such as minimum and maximum length values, which we need access to in order to build error messages such as `Your password needs to be at least <n> characters long.`.

Reply:
After this change would a developer have to pass the config prop for this to work as expected ? I don't advocate for devs to have to pass `Clerk.__unstable__environment.userSettings.passwordSettings` but it seems that this can be abstracted away as element already have access to environment internally. 

Saying this because i saw a PR for docs, and i'm not sure why we are thinking this should be public API

Reply:
`config` was already being destructured in the `usePassword` hooks for use in the `generatePasswordErrorText` usage within elements. This PR is just exposing the usage of `config` within fieldstate for users to be able to use and translate the errors in usage of elements.

We are not requiring devs to access `Clerk.__unstable__environment.userSettings.passwordSettings` if that is what you are asking.

Reply:
If there is a concern about exposing the config here, we can remove and access the config within UI package via `useClerk` and circle back and figure out the best solution.

Reply:
Maybe I've not understood the usage of config here. I don't want to block this. Is this config for providing the text that will be used for localization or is this config used for (i.e.) setting the minimum password length ?

Reply:
@panteliselef no worries, appreciate the review. You can see the config usage here in our UI usage of elements https://github.com/clerk/javascript/pull/3812/files#diff-44ae29a5976ef7ae53f3bda6592ff474783e38e79d777b25066f0b664be58cf4R307